### PR TITLE
Remove redundant parentheses from Python files

### PR DESCRIPTION
**Pull request by jbampton**

### CODE REVIEW

File: airflow/migrations/versions/952da73b5eff_add_dag_code_table.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ class SerializedDagModel(Base):
                     sa.Column('last_updated', sa.TIMESTAMP(timezone=True), nullable=False))
 
     conn = op.get_bind()
-    if conn.dialect.name not in ('sqlite'):
+    if conn.dialect.name not in 'sqlite':
```
Comment (plaes) [inline airflow/migrations/versions/952da73b5eff_add_dag_code_table.py]: It seems that original code was actually wrong:
```
'sql' in 'sqlite' => True
'sql' in ('sqlite',) => False
```
So, original code should actually be: `.. not in ('sqlite',)`:

File: airflow/providers/exasol/hooks/exasol.py [hunk 2]
```diff
@@ -145,7 +145,7 @@ def set_autocommit(self, conn, autocommit):
         """
         if not self.supports_autocommit and autocommit:
             self.log.warning(
-                ("%s connection doesn't support " "autocommit but autocommit activated."),
+                "%s connection doesn't support " "autocommit but autocommit activated.",
```
Comment (mik-laj) [inline airflow/providers/exasol/hooks/exasol.py]:
```suggestion
                "%s connection doesn't support autocommit but autocommit activated.",
```

File: airflow/migrations/versions/952da73b5eff_add_dag_code_table.py [hunk 3]
```diff
@@ -59,7 +59,7 @@ class SerializedDagModel(Base):
                     sa.Column('last_updated', sa.TIMESTAMP(timezone=True), nullable=False))
 
     conn = op.get_bind()
-    if conn.dialect.name not in ('sqlite'):
+    if conn.dialect.name not in ('sqlite',):
```
Comment (plaes) [inline airflow/migrations/versions/952da73b5eff_add_dag_code_table.py]: This should be a separate commit.

**Comment by jbampton** [comment]
Hey @potiuk can you please review 👍

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Thanks @jbampton !
